native/app.py

```
@@ -0,0 +1,154 @@
+@app.route("/api/groups/<name>", methods=["DELETE"])
+def delete_group(name):
+    res = mcpj("delete", "group", name)
+    f = GROUPS_DIR / f"{name}.json"
+    f.unlink(missing_ok=True)
+    return jsonify({"name": name, "stdout": res.stdout.strip()})
+
+
+if __name__ == "__main__":
+    GROUPS_DIR.mkdir(parents=True, exist_ok=True)
+    CONFIGS_DIR.mkdir(parents=True, exist_ok=True)
+    app.run(host="0.0.0.0", port=PORT)
```

Ruff (0.15.12)

[error] Possible binding to all interfaces

(S104)